XTS/recovery/src/org/jboss/jbossts/xts/recovery/participant/ParticipantRecoveryInitialisation.java

```diff
@@ -65,16 +66,20 @@ public static void startup()
             }
             
             try {
-                XTSRecoveryModule module = (XTSRecoveryModule)clazz.newInstance();
+                XTSRecoveryModule module = (XTSRecoveryModule)clazz.getDeclaredConstructor().newInstance();
                 module.install();
                 RecoveryManager.manager().addModule(module);
                 recoveryModules.add(module);
             } catch (InstantiationException ie) {
                 RecoveryLogger.i18NLogger.error_recovery_participant_ParticipantRecoveryInitialisation_3(className, ie);
             } catch (IllegalAccessException iae) {
                 RecoveryLogger.i18NLogger.error_recovery_participant_ParticipantRecoveryInitialisation_4(className, iae);
-            }
-        }
+            } catch (InvocationTargetException e) {
+                RecoveryLogger.i18NLogger.error_recovery_participant_ParticipantRecoveryInitialisation_4(className, e);
+	    } catch (NoSuchMethodException e) {
+                RecoveryLogger.i18NLogger.error_recovery_participant_ParticipantRecoveryInitialisation_4(className, e);
+	    }
+	}
```

Comment:
`error_recovery_coordinator_CoordinatorRecoveryInitialisation_4` is used to report `Unable to access module class`...I don't think you can reuse it for these other exceptions

Reply:
@jmfinelli here i think for `NoSuchMethodException` we can use "Unable to find module class {0}" and for `InvocationTargetException` we can use "Unable to invoke module class {0}"